content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,321 @@
+  respectively).
+- :guilabel:`Journal`: select the sales electronic journal.
+- :guilabel:`Document Type`: type document type in this format, for example, `(111) e-Invoice`.
+- :guilabel:`Products`: specify the product with the correct taxes.
+
+Customer credit note
+~~~~~~~~~~~~~~~~~~~~
+
+The :doc:`Customer credit note <../accounting/customer_invoices/credit_notes>` is an electronic
+document that, when validated, is sent to |DGI| via Uruware. It is necessary to have a validated
+(posted) invoice to register a credit note. On the invoice there is a button named :guilabel:`Credit
+note`, click on this button to be directed to the :guilabel:`Create credit note` form, then complete
+the following information:
+
+- :guilabel:`Reason`: type the reason for the credit note.
+- :guilabel:`Journal`: select the journal that has to be electronic and has the :guilabel:`Use
+  Documents?` option active.
+- :guilabel:`Document Type`: select the credit note document type.
+- :guilabel:`Reversal Date`: type the date.
+
+Customer debit note
+~~~~~~~~~~~~~~~~~~~
+
+The :doc:`Customer debit note <../accounting/customer_invoices/credit_notes>` is an electronic
+document that, when validated, is sent to |DGI| via Uruware. It is necessary to have a validated
+(posted) invoice to register a debit note. On the action within an invoice there is a option named
+:guilabel:`Debit note`, click on this action to be directed to the :guilabel:`Create credit note`
+form, then complete the following information:
+
+- :guilabel:`Reason`: type the reason for the credit note.
+- :guilabel:`Journal`: select the journal that has to be electronic and has the :guilabel:`Use
+  Documents?` option active.
+- :guilabel:`Copy lines`: mark checkbox to copy the invoice lines to the debit note.
+- :guilabel:`Debit note date`: type the date.
+
+.. note::
+   The document sequence (number) is brought from Uruware once the document has been processed.
+
+.. note::
+   The PDF of the validated document is pulled from Uruware following the specification by the
+   Uruguayan government (DGI).
+
+Addendas and disclosures
+========================
+
+*Addendas* and *disclosures* are additional notes and comments added to an electronic document that
+can be mandatory or optional. To create a new addenda, go to :menuselection:`Accounting --> Settings
+--> DGI --> Addendas and disclosures`.
+
+To create an addenda, the following information is required:
+
+- :guilabel:`Name`: Name of the Addenda or Mandatory Disclosure.
+- :guilabel:`Apply on`: Select the documents where the addenda or mandatory disclosure are
+  applicable.
+- :guilabel:`Type`: Select the type of remark, this will add it to the specific section in the XML.
+- :guilabel:`Content`: Add the complete text of the addenda or disclosure.
+- :guilabel:`Is disclosure?`: Select this box if the text is a mandatory disclosure, leave it blank
+  if it is additional information.
```

Comment:
We removed the "Apply on" filed. Let's remove it from here too. Also the "Is disclosure" field changed to "Is Legend" and goes before the content.